examples/src/main/scala/org/hyperscala/examples/ui/SocialSharingExample.scala

```diff
@@ -0,0 +1,29 @@
+package org.hyperscala.examples.ui
+
+import org.hyperscala.examples.Example
+import org.hyperscala.html.StaticHTML
+import org.hyperscala.ui.module.SocialSharing
+import org.hyperscala.ui.module.SocialSharing.MessageData
+import org.powerscala.IO
+import org.hyperscala.html._
+
+/**
+ * Created by mmynsted on 4/6/15.
+ * code@growingliberty.com
+ */
+class SocialSharingExample extends Example {
+  this.require(SocialSharing)
+
+  val messageData: MessageData = Map("title" -> "Check out Hyperscala.com",
```

Comment:
Using a Map is bad design. The data is rigidly defined, so it should really just be a case class.